resource_management/MessagePriority priority
bool data
